tracer/src/Datadog.Trace/Iast/Aspects/AspNetCore.Http/HttpResponseAspect.cs

```diff
@@ -25,6 +25,11 @@ public class HttpResponseAspect
     [AspectMethodInsertBefore("Microsoft.AspNetCore.Http.HttpResponse::Redirect(System.String,System.Boolean)", 1)]
     public static string? Redirect(string? url)
     {
-        return IastModule.OnUnvalidatedRedirect(url);
+        if (Iast.Instance.Settings.Enabled)

```

Comment:
I'd move this checks to the `IastModule` instead of adding the guard on each aspect

Reply:
I thought about it, but I preferred this option mainly for performance reasons, to avoid calling a method and returning IastModuleResponse.Empty each time.

Also, it can combine well with some aspects that already check some conditions within the aspect, such as MongoDatabaseAspect.cs, HttpClientAspect.cs or HttpSessionStateBaseAspect.cs